README.md

```diff
@@ -9,55 +9,108 @@ http://52.24.131.202:8999/
 
 **NPM:** https://www.npmjs.com/package/gusto-stratera
 
+1)
+Assigned one leader, gave that person authority and responsibility, and held that person accountable for the quality of the prototype submitted.
 
-## Dependencies
-This is a node.js application which requires you to have node installed. To install node, please visit ```https://nodejs.org/download/``` and download and run the latest version of node.
+[See email](DevelopmentPoolEvidence/team/TechnicalLeadEmail.JPG)
 
-Alternatively, it's available for installation through a number of package managers, for example:
+2)
+Assembled a multidisciplinary and collaborative team that includes at a minimum of two of the labor categories limited to the Development Pool labor categories.    
 
-*chocolatey:*
-```bash
-choco install nodejs.install
-```
+Our Team consists of a DevOps Engineer and a Back-end Developer.  Both team members are experienced in the agile scrum development methodology and in playing the role of scrum master.  The DevOps Engineer plays the role of Scrum Master and also performs some of the development tasks.  The Back-end Developer performs the development quality assurance tasks such as peer reviews.
 
-*yum:*
-```bash
-sudo yum install nodejs npm --enablerepo=epel
-```
+3)
+Used at least five modern and open-source technologies, regardless of architectural layer (frontend, backend, etc.)
 
-Other package manager examples can be found on [joyent's github page](https://github.com/joyent/node/wiki/Installing-Node.js-via-package-manager "joyent's github page").
- 
+| Technology | Version |
+| ---------- | ------- |
+| centos | 6.5 |
+| node | 0.12.0 |
+| express | 4.12.3 |
+| express-http-proxy | 0.6.0 |
+| istanbul | 0.3.16 |
+| mocha | 2.1.0 |
+| mocha-teamcity-reporter | 0.0.1 |
+| should | 7.0.1 |
+| supertest | 1.0.1 |
+| git |1.9.5 | 
```

Comment:
Good as long as long as five of the technologies or standards released, created, initiated, or finalized in the 5 years preceding the release of the RFQ.  It must also be Open Source.  Just wanted to verify